orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -304,7 +307,17 @@ def _get_exported_model(self, input_schema: _ModelInputOutputSchemaType, *inputs
 
         TODO: How to support dynamic axes? Dimensions are determined by samples
         """
-        with _logger.suppress_os_stream_output(log_level=self._debug_options.logging.log_level) as suppress_output:
+
+        # VERBOSE -> FULL export verbose log + FULL torch other logs from stdout and stderr (C++ backend)
+        # INFO -> FULL export verbose log + FILTERED torch other logs from stdout and stderr (C++ backend)
+        # WARNING/ERROR -> [Rank 0] NO export verbose log + FILTERED torch other logs from stdout and stderr (C++ backend)
+        # Be noted: rank 0 log only is controlled by logger configured in _logger.py
+        torch_exporter_verbose_log = self._debug_options.logging.log_level <= LogLevel.INFO
+        print_on_exit = _logger.create_log_filter(
+            self._logger, self._debug_options.torch_exporter_filter, "model export to ONNX"
+        )
+        self._logger.warning("Exporting the PyTorch model to ONNX...")
```

Comment:
Is warning necessary here? It seems more like an info message.

Reply:
moved to info. previously I was thinking if export takes too long, we'd better let users know it is enter exporting phase.